### PR TITLE
Create ba.lua

### PR DESCRIPTION
**Pull request by katotekii**
here you go luke

### CODE REVIEW

File: data/locales/ba.lua [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+	['ui_removeattachments'] = "Skini Dodatke",
+	-- Tooltip
+	['ui_durability'] = "Izdržljivost",
+	['ui_ammo'] = "Metci",
+	['ui_serial'] = "Serijski Broj",
+	['ui_components'] = "Komponente",
+	['ui_tint'] = "Tinta",
+	-- Useful controls
+	['ui_usefulcontrols'] = "Korisne kontrole",
+	['ui_rmb'] = "Item Contex Menu",
```
Comment (LukeWasTakenn) [inline data/locales/ba.lua:16]: Perhaps instead of this just do "Item kontekst meni"?

File: data/locales/ba.lua [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+	['ui_ctrl_lmb'] = "Brzo premjestite hrpu predmeta u drugi inventar",
```
Comment (LukeWasTakenn) [inline data/locales/ba.lua:17]: Here you use "predmeta" but in most cases you use "item" so should really decide between the two

I think "predmet" is probably a better translation all around

File: data/locales/ba.lua [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+	['ui_shift_drag'] = "Razdvoji na pola iteme",
```
Comment (LukeWasTakenn) [inline data/locales/ba.lua:18]: Should be "Razdvoji iteme na pola"

**Review by LukeWasTakenn — CHANGES_REQUESTED** [review]

**Review by katotekii — COMMENTED** [review]
here you go

**Comment by LukeWasTakenn** [comment]
Hvala punoo

**Pull request merged**